File: docs/components_page/components/dropdown/alignment.R
library(dashBootstrapComponents)

items <- list(
  dbcDropdownMenuItem("Action"),
  dbcDropdownMenuItem("Another action"),
  dbcDropdownMenuItem("Something else here"),
  dbcDropdownMenuItem(divider = TRUE),
  dbcDropdownMenuItem("Something else here after the divider")
)

dropdown <- dbcRow(
  list(
    dbcCol(
      dbcDropdownMenu(
        label = "Left-aligned menu (default)",
        children = items,
        align_end = FALSE
      )
    ),
    dbcCol(
      dbcDropdownMenu(
        label = "Right-aligned menu", children = items, align_end = TRUE
      )
    )
  )
)

File: docs/components_page/components/dropdown/alignment.jl
using DashBootstrapComponents

items = [
    dbc_dropdownmenuitem("Action"),
    dbc_dropdownmenuitem("Another action"),
    dbc_dropdownmenuitem("Something else here"),
    dbc_dropdownmenuitem(divider = true),
    dbc_dropdownmenuitem("Something else here after the divider"),
];

dropdown = dbc_row([
    dbc_col(
        dbc_dropdownmenu(
            label = "Left-aligned menu (default)",
            children = items,
            align_end = false,
        ),
    ),
    dbc_col(
        dbc_dropdownmenu(label = "Right-aligned menu", children = items, align_end = true),
    ),
]);
